modules/installation-uninstall-clouds.adoc

```diff
@@ -48,10 +48,49 @@ endif::gcp[]
 cluster.
 ifdef::ibm-cloud[]
 * You have configured the `ccoctl` binary.
+* You have installed the IBM Cloud CLI and installed or updated the VPC infrastructure service plug-in. For more information see "Prerequisites" in the link:https://cloud.ibm.com/docs/vpc?topic=vpc-infrastructure-cli-plugin-vpc-reference&interface=ui#cli-ref-prereqs[IBM VPC CLI documentation].
 endif::ibm-cloud[]
 
 .Procedure
+ifdef::ibm-cloud[]
```

Comment:
@cjschaef Please take a pass at this procedure. In particular, can you define what `<infrastructure_id>` is? Thanks in advance.